docs/pages/database-access/guides/rds.mdx

```diff
@@ -1,6 +1,6 @@
 ---
-title: Database Access with AWS RDS and Aurora for PostgreSQL and MySQL
-description: How to configure Teleport Database Access with AWS RDS and Aurora for PostgreSQL and MySQL.
+title: Database Access with AWS RDS and Aurora for PostgreSQL, MySQL and MariaDB.
```

Comment:
```suggestion
title: Database Access with AWS RDS and Aurora for PostgreSQL, MySQL and MariaDB
```

Reply:
Also, the `title` is now long enough to be truncated in Google's search results. To prevent this, let's add an `h1` frontmatter item with this title—this will just be used for presentation within the page—and give `title` the value, `Database Access with AWS RDS and Aurora`.